hydromt_sfincs/workflows/merge.py

```diff
@@ -88,9 +88,20 @@ def merge_multi_dataarrays(
         method = "bilinear"
 
     if da_like is not None:  # reproject first raster to destination grid
-        da1 = da1.raster.reproject_like(da_like, method=method).load()
+        da1 = da1.raster.clip_bbox(
+            da_like.raster.box.to_crs(da1.raster.crs.to_epsg()).total_bounds, buffer=10
+        )
+        if np.any(np.array(da1.shape) <= 2):
+            # no data in da1 so use an empty array like da_like
+            logger.debug("No data da1, start with nan array")
+            da1 = xr.full_like(da_like, np.nan)
+        else:
+            da1 = da1.load()
+            da1 = da1.raster.reproject_like(da_like)
     elif reproj_kwargs:
-        da1 = da1.raster.reproject(method=method, **reproj_kwargs).load()
+        #TODO also clip to bbox?
```

Comment:
I don;t think this option is used in the code anymore .. However, if we want to koop it, maybe we have to add the clipping to the bbox here as well?

Reply:
see also comment below